### PR TITLE
Add docs for changing log levels dynamically

### PR DESCRIPTION
**Pull request by deepthidevaki**
## Description

Add docs on how to change log levels dynamically.

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

**Review by pihme — CHANGES_REQUESTED** [review]
This looks all good and well. 

However, I think we should also mention it in the Zeebe documentation in the Operations Chapter

Maybe the best place with the current content would be to add this also to operations/configuration.md/Logging

**Comment by deepthidevaki** [comment]
Thanks @pihme . Docs are now added to the main documentation. Please have a look.

File: docs/src/operations/configuration.md [hunk 1]
```diff
@@ -145,6 +145,17 @@ To enable Google Stackdriver compatible JSON logging you can set the environment
 {{#include ../../../dist/src/main/config/log4j2.xml}}
 ```
 
+### Change log level dynamically
+
+Zeebe brokers expose a [Spring Boot Actuators web endpoints](https://docs.spring.io/spring-boot/docs/current/actuator-api/html/#loggers)
```
Comment (pihme) [inline docs/src/operations/configuration.md:150]:
```suggestion
Zeebe brokers expose a [Spring Boot Actuators web endpoint](https://docs.spring.io/spring-boot/docs/current/actuator-api/html/#loggers)
```

File: docs/src/operations/configuration.md [hunk 2]
```diff
@@ -145,6 +145,17 @@ To enable Google Stackdriver compatible JSON logging you can set the environment
...
+ for configuring loggers dynamically. 
+To change the log level of a broker make a `POST` request to the `/actuator/loggers/{logger.name}` endpoint as shown in the example below.
```
Comment (pihme) [inline docs/src/operations/configuration.md:152]:
```suggestion
To change the log level of a logger make a `POST` request to the `/actuator/loggers/{logger.name}` endpoint as shown in the example below.
```

**Review by pihme — APPROVED** [review]
Some editorial changes

**Pull request merged**